event_routing_backends/handlers.py

```diff
@@ -32,6 +33,9 @@ def send_tracking_log_to_backends(
     This allows us to only send the tracking log to the event bus once and the event bus will send
     the transformed event to the different configured backends.
     """
+    if not kwargs.get(SIGNAL_PROCESSED_FROM_EVENT_BUS, False):
+        logger.info("Event received from a non-event bus backend, skipping...")
```

Comment:
I think this will get very spammy in cases like the tracking events where this is an expected condition. Maybe make it a `logger.debug`?